pkg/printers/internalversion/printers_test.go

```diff
@@ -6601,7 +6601,7 @@ func TestPrintIPAddressList(t *testing.T) {
 	// Columns: Name, ParentRef, Age
 	expected := []metav1.TableRow{
 		{Cells: []interface{}{"192.168.2.2", "myresource.mygroup/mynamespace/myname", "10y"}},
```

Comment:
change this one as well so we don't hit this issue again

Reply:
Done.